docker-compose/standalone-d2cc/docker-compose.yml

```diff
@@ -249,6 +249,7 @@ services:
     image: farao/gridcapa-cse-adapter:latest
     volumes:
       - ../config/cse/d2cc/cse-d2cc-gridcapa-adapter-application.yml:/config/application.yml
+      - ../config/cse/d2cc/20211209_Targets_CH.xml:/config/target-ch.xml
```

Comment:
@TheMaskedTurtle  please update the docker-compose file in the acceptance-tests package

Reply:
done